docker/docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: '3'
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)